packages/EasyCore/src/Bridge/Laravel/Middleware/TrimStrings.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Bridge\Laravel\Middleware;
+
+use EonX\EasyCore\Tests\Helpers\CleanerInterface;
+use Illuminate\Http\Request;
+use Symfony\Component\HttpFoundation\ParameterBag;
+
+final class TrimStrings
+{
+    /**
+     * The attributes that should not be trimmed.
+     *
+     * @var mixed[]
+     */
+    private $except;
+
+    /**
+     * @var CleanerInterface
+     */
+    private $cleaner;
+
+    public function __construct(CleanerInterface $cleaner, array $except = [])
+    {
+        $this->cleaner = $cleaner;
+        $this->except = $except;
+    }
+
+    /**
+     * Handle an incoming request.
```

Comment:
Let's remove descriptions here and for other methods too. The methods are simple and obvious.

Reply:
Done.